pages/welcome/sections/user_group_section.py

```diff
@@ -0,0 +1,174 @@
+                                            src="assets/welcome_user_group_section/new_group_name.png",
+                                        )
+                                    ],
+                                ),
+                            ],
+                        ),
+                        html.Div(
+                            className="instruction_card instruction_card_split",
+                            children=[
+                                html.Div(
+                                    className="instruction_card instruction_card_body",
+                                    children=[
+                                        html.H2("3. Add Repositories and Organizations"),
+                                        html.P(
+                                            """
+                                                Once you add your new group, you can now select it in the "group
+                                                name" dropdown. Once selected, now add the repositories and organizations
+                                                you want in your group and click "add". Note: if the repository or organization
+                                                does not already exist in the database, adding it to a group will trigger
+                                                collection but will not automatically show up in 8Knot. To check if the repository
+                                                does not already exist in Augur, it will show as all "0s" in the Augur front end.
+                                            """
+                                        ),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="instruction_card instruction_card_body centered_img",
+                                    children=[
+                                        html.Img(
+                                            className="instruction_image",
+                                            src="assets/welcome_user_group_section/add_repos.png",
+                                        )
+                                    ],
+                                ),
+                            ],
+                        ),
+                        html.Div(
+                            className="instruction_card instruction_card_split",
+                            children=[
+                                html.Div(
+                                    className="instruction_card instruction_card_body",
+                                    children=[
+                                        html.H2("4. Click Refresh Groups"),
+                                        html.P(
+                                            """
+                                                Once you add your new group, go back to the 8Knot front end and click
```

Comment:
imo 'Once you have created or edited your groups, go back to 8Knot and click...'